source-code/ide-extension/src/decorations/contextTooltip.ts

```diff
@@ -0,0 +1,43 @@
+import type { MessageReferenceMatch } from '@inlang/core/config';
+import { query } from '@inlang/core/query';
+import { MarkdownString, Uri } from 'vscode';
+import { state } from '../state.js';
+
+const MISSING_TRANSLATION_MESSAGE = '[missing]'
+
+type ContextTableRow = {
+  language: string
+  message: string
+  editCommand?: Uri
+  openInEditorCommand?: Uri
+}
+
+function renderTranslationRow(row: ContextTableRow) {
+  const messageListing = `<td><strong>${row.language}</strong></td><td>${row.message}</td>`
+  const editCommand = row.editCommand ? `<td><a href="${row.editCommand}">$(edit)</a></td>` : ''
+  const openInEditorCommand = row.openInEditorCommand ? `<td><a href="${row.openInEditorCommand}">$(link-external)</a></td>` : ''
+  return `<tr>${messageListing}${editCommand}${openInEditorCommand}</tr>`
```

Comment:
@openscript Can we get a littlebit of whitespace in the table cells? 

For example, additional 2px on each side of the column and 4px on the outside of the table? I recognized that the window easily closes (that was a concern we decided to go with) and it could probably be a bit less jumpy if we increase the space in the tooltip by a littlebit.

Reply:
Apart from `&nbsp;` I was not able to take an influence on the padding. I think the styling options are limited (for a good reason in my opinion).

Before:
![image](https://github.com/inlang/inlang/assets/1105080/df4467b3-92fd-4881-84b9-c4506edc1676)

After:
![image](https://github.com/inlang/inlang/assets/1105080/2e8be28d-2284-48a0-80d1-fd4cbc842f1b)

The tooltip looks better to me, even though I consider `&nbsp;` as an ugly hack.

Reply:
@openscript That makes it much better in my opinion! Let's stay with the hack for now and see how the tooltip envolves.